### PR TITLE
chore: add multiple issue templates

### PR DESCRIPTION
**Pull request by ncoden**
Add the following issue templates:
* 🐛 **Bug Report**
          Something is broken or doesn't work as expected
* ✨ **Feature Request**
          Something is missing or could be improved
* 📖 **Documentation**
          Typo mistakes, gaps or inconsistencies in docs or website
* 💻 **Maintainance**
          Tests, dependencies, workflow...

Closes https://github.com/zurb/foundation-sites/issues/11241

### CODE REVIEW

**Comment by ncoden** [comment]
Poke @DanielRuf for review :)

**Comment by ncoden** [comment]
Poke @DanielRuf for review :)

File: .github/ISSUE_TEMPLATE/1-bug-report.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+<!---                 Please fill the following template                    -->
+<!---                 Your issue may be ignored otherwise                   -->
+<!--- --------------------------------------------------------------------- -->
+<!---  Only submit bug reports here. For help or questions to the community -->
+<!---  see the forum: https://foundation.zurb.com/forum                     -->
+
+## What should happen?
+
+
+## What happends instead?
```
Comment (DanielRuf) [inline .github/ISSUE_TEMPLATE/1-bug-report.md]: happends => happens

File: .github/ISSUE_TEMPLATE/2-feature-request.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+<!---                 Your issue may be ignored otherwise                   -->
+<!--- --------------------------------------------------------------------- -->
+<!---  Only submit feature requests here. For help or questions to the      -->
+<!---  community, see the forum: https://foundation.zurb.com/forum          -->
+
+## What is missing?
+<!--- What you want to be added or improved and how you would use it?       -->
+
+## Why is it important?
+<!--- Why would this feature benefits to everyone? What is the current      -->
```
Comment (DanielRuf) [inline .github/ISSUE_TEMPLATE/2-feature-request.md]: benefits to => benefit

File: .github/ISSUE_TEMPLATE/4-maintainance.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+name: "\U0001F4BB Maintainance"
+about: Tests, dependencies, workflow...
+
+---
+
+<!--- --------------------------------------------------------------------- -->
+<!---                 Please fill the following template                    -->
+<!---                 Your issue may be ignored otherwise                   -->
+<!--- --------------------------------------------------------------------- -->
+<!---  Only submit maintainance issue here. For help or questions to the.   -->
```
Comment (DanielRuf) [inline .github/ISSUE_TEMPLATE/4-maintainance.md]: issue => issues

**Review by DanielRuf — APPROVED** [review]

**Review by DanielRuf — CHANGES_REQUESTED** [review]
Some small typos.

**Comment by ncoden** [comment]
@DanielRuf Fixed

**Comment by ncoden** [comment]
poke @DanielRuf

**Review by DanielRuf — APPROVED** [review]

**Pull request merged**